paddle/fluid/operators/ngraph/ngraph_engine.h

```diff
@@ -127,9 +129,7 @@ class NgraphEngine {
 
   void Run(const framework::Scope& scope, const platform::Place& place) const;
 
-  static bool is_training;
-  static const framework::BlockDesc* p_bdesc;
-  static std::vector<std::string> feed_vars, fetch_vars;
+  static std::vector<std::string> feed_vars;
```

Comment:
why only keep feed vars as static?

Reply:
It is unlikely to cause issue, so keep it for now. It can be removed if desired